uaaextras/tests.py

```diff
@@ -1020,6 +1057,19 @@ def test_fail_to_render_temporary_password_no_token(
             render_template.assert_called_with("reset_password.html")
             flash.assert_called_once()
 
+    @requests_mock.Mocker(kw="m")
+    def test_bad_token_redirects(self, m):
+        """The session is cleared when logging out"""
+
+        m.post("https://uaa.bosh-lite.com/introspect", text='{"active": false}')
+        with app.test_request_context("/logout"):
+            with app.test_client() as c:
+                with c.session_transaction() as sess:
+                    sess["UAA_TOKEN"] = "foo"
+
+                rv = c.get("/")
+                assert rv.status_code == 302
+
```

Comment:
this tests the new functionality. I didn't add an explicit test for valid tokens because basically every other test is testing this